bukkit/src/main/java/co/aikar/commands/BukkitCommandIssuer.java

```diff
@@ -82,8 +82,8 @@ public boolean hasPermission(String name) {
 
     @Override
     public boolean equals(Object o) {
-        if (this == o) return true;
-        if (o == null || getClass() != o.getClass()) return false;
+        if (this == o) { return true; }
```

Comment:
This is the case where I hate 1TBS, wouldn't it be better to use K&R for this kind of stuff? In my opinion, this looks a lot better:
```
if (this == o)
    return true;
```
@aikar thoughts on this?

Reply:
I'll change it around to allow for non-oneliners if either @aikar or another member could provide some more insights on their view on this. I'd still like to comply with Google's standard of using braces, though:
```java
if (this == o) {
    return true;
}
```

Reply:
I would use `if (this == o) return true;` over the other solutions proposed here

Reply:
In that case, I'd like to wait for either aikar or another member to take a go before I change anything in the style guide on the PR.

Reply:
I've asked for @bramhaag's insights, and he said as follows:
```
[10:53] bramhaag: whatever you do
[10:53] bramhaag: don't do if (this == o) { return true; }
[10:54] bramhaag: 
if (this == o) {
    return true;
}
 for consistency
[10:54] bramhaag: or 
if (this == o)
    return true;
 for aesthetics
```

I guess it boils down to those two at last, so I'd like to simply propose to modify Google's standard to use the second ("aesthetically appealing") version.

Reply:
I prefer braces but def dont wanna do 1 line braces,

if () {
 }

please.

This was all generated by IDE is why it ended up like that.